Installing-and-Configuring/Upgrading.md

```diff
@@ -31,7 +28,7 @@ Recovery shell
 
 If the flash protection bits are set correctly it is not possible to
 rewrite the firmware from the normal OS.  You'll need to reboot
-to the Heads recovery shell (hit `r` after the TPM TOTP prompt).
+to the Heads [recovery shell](/RecoveryShell/). Hold 'r' on boot or choose 'Recovery Shell' in the heads GUI.
```

Comment:
duplicate. hit r frenetically, not just hold r

Reply:
fixed.  will be pushed in a few minutes